content/news/2022-12-23/122/index.ru.md

```diff
@@ -6,7 +6,7 @@ slug: "we-wish-you-a-merry-christmas-and-a-happy-new-year"
 
 Мы поздравляем вас с Рождеством 🎅 и Новым годом 🎄! Лучший подарок для нас — это знать, что мы делаем что-то нужное и полезное. Встречайте наш маленький подарок 🎁 вам: новогодний релиз Organic Maps доступен в [AppStore](https://apps.apple.com/app/organic-maps/id1567437057), [Google Play](https://play.google.com/store/apps/details?id=app.organicmaps), Huawei [AppGallery](https://appgallery.huawei.com/#/app/C104325611?local=en) и [FDroid](https://f-droid.org/en/packages/app.organicmaps/)!
 
-На данный момент на нашем GitHub](https://github.com/organicmaps/organicmaps/issues) более 1200 ошибок, которые надо исправить, и новых функций, которые хотелось бы реализовать. Многие из них направлены на улучшение приложения. Некоторые возможности более сложны и могут быть реализованы как отдельные, независимые открытые проекты. Например, фотографии мест и [отзывы](https://github.com/organicmaps/organicmaps/issues/2758), база данных [дорожного движения](https://github.com/organicmaps/organicmaps/issues/1160), сервис [общественного транспорта](https://github.com/organicmaps/organicmaps/issues/837), [облачная синхронизация и резервное копирование](https://github.com/organicmaps/organicmaps/issues/2082), онлайн-карты, веб-редактор для планирования поездок, редактор стилей карт, [путеводители](https://github.com/organicmaps/organicmaps/issues/3648), API/SDK для других приложений и т.д. Вы можете помочь [финансово](https://organicmaps.app/donate/), либо учавствуя или даже возглавляя такой сложный проект!
+На данный момент на нашем GitHub](https://github.com/organicmaps/organicmaps/issues) более 1800 ошибок, которые надо исправить, и новых функций, которые хотелось бы реализовать. Многие из них направлены на улучшение приложения. Некоторые возможности более сложны и могут быть реализованы как отдельные, независимые открытые проекты. Например, фотографии мест и [отзывы](https://github.com/organicmaps/organicmaps/issues/2758), база данных [дорожного движения](https://github.com/organicmaps/organicmaps/issues/1160), сервис [общественного транспорта](https://github.com/organicmaps/organicmaps/issues/837), [облачная синхронизация и резервное копирование](https://github.com/organicmaps/organicmaps/issues/2082), онлайн-карты, веб-редактор для планирования поездок, редактор стилей карт, [путеводители](https://github.com/organicmaps/organicmaps/issues/3648), API/SDK для других приложений и т.д. Вы можете помочь [финансово](https://organicmaps.app/donate/), либо учавствуя или даже возглавляя такой сложный проект!
```

Comment:
There is no need to change the number in the old news, let's keep it for the grow history. Now we can say that we have +600 issues in 7 months )

Reply:
I just replaced all `1200` results by `1800`, so making this distinction makes sense.